django/santropolFeast/santropolFeast/templates/base.html

```diff
@@ -5,7 +5,7 @@
 <head>
     <meta charset="UTF-8">
     <title>
-        {% block title %}Feast On Wheels{% endblock %}
+        Feast On Wheels
```

Comment:
Becarefull, here we change a file who have no contact with your issues.

However, if you delete the ```block title``` we can't change the title for each page.

*In UI have a same title for all page it's a bad practice with no value added for visitors.*
A title by page give more information when you read your tab in your browser and help you to switch between multiple tab of a same website.